src/Sunset/EventListener/AddSunsetHeaderListener.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Sunset\EventListener;
```

Comment:
Just wondering why not directly in Core\Eventlistener and I faill to see where is the listener declared (src/Bridge/Symfony/Bundle/Resources/config/api.xml) ?

Reply:
Good catch. Should be in core, and of course registered, thank @antograssiot

Reply:
So i moved it to EventListener\RespondListner::onKernelView(), was that what you had in mind @antograssiot ?